.github/issues_labeler.yml

```diff
@@ -0,0 +1,32 @@
+APIs:
+  - "(api|APIs|apis|API|APIS)"
+BASH:
+  - "(Bash|bash|BASH)"
+javascript:
+  - "(JavaScript|JAVASCRIPT|Javascript|javascript|JS|js|Js)"
+python:
```

Comment:
We don't thave anything related to python in this repo, same goes for Bash and APIs

Reply:
Well, that's kind of optional. I can remove it if you want.